llama.cpp

```diff
@@ -4423,8 +4423,8 @@ static void llm_load_vocab(
                 vocab.type_pre = LLAMA_VOCAB_PRE_TYPE_STARCODER;
             } else if (
                     tokenizer_pre == "gpt-2"   ||
-                    tokenizer_pre == "jina-es" ||
-                    tokenizer_pre == "jina-de") {
+                    tokenizer_pre == "jina-v2-es" ||
+                    tokenizer_pre == "jina-v2-de") {
```

Comment:
This would be a breaking change for anyone who started using the Jina models. If you think the impact is small, we can merge like this. Otherwise, we can keep the old versionless strings for a while to guarantee smooth transition

Reply:
let me add both as in the `||`